etl/processor/processor.go

```diff
@@ -0,0 +1,98 @@
+package processor
+
+import (
+	"context"
+
+	"github.com/insolar/block-explorer/etl/interfaces"
+	"github.com/insolar/block-explorer/etl/models"
+	"github.com/insolar/block-explorer/etl/types"
+)
+
+type Processor struct {
+	JDC     <-chan types.JetDrop
+	TaskC   chan Task
```

Comment:
Why create `TaskC` channel and not just use `JDC`?

Reply:
To have control level between